default-with-analytics/WSO2AM210-All-In-One-Deployment-VPC.template.yaml

```diff
@@ -646,172 +605,40 @@ Resources:
-      DBInstanceIdentifier: WSO2APIMAnalyticsDBInstance
-      DBName: WSO2APIMAnalytics_DB
-      Engine: MySQL
-      MasterUsername: !Ref DBUsername
-      MasterUserPassword: !Ref DBPassword
-      MultiAZ: 'true'
-      StorageType: gp2
-      DBSubnetGroupName: !Ref WSO2APIMDBSubnetGroup
-  WSO2APIMDBSubnetGroup:
-    Type: 'AWS::RDS::DBSubnetGroup'
-    Properties:
-      DBSubnetGroupDescription: DB Subnet Group
-      SubnetIds:
-        - !Ref WSO2APIMPrivateSubnet1
         - !Ref WSO2APIMPrivateSubnet2
-  WSO2APIMEFSFileSystem:
-    Type: 'AWS::EFS::FileSystem'
-  WSO2APIMAnalyticsEFSFileSystem:
-    Type: 'AWS::EFS::FileSystem'
-  WSO2APIMPrivateSubnetEFSMountTarget:
-    Type: 'AWS::EFS::MountTarget'
-    Properties:
-      SubnetId: !Ref WSO2APIMPrivateSubnet1
-      FileSystemId: !Ref WSO2APIMEFSFileSystem
-      SecurityGroups:
-        - !Ref WSO2APIMEFSSecurityGroup
-  WSO2APIMPublicSubnetEFSMountTarget:
-    Type: 'AWS::EFS::MountTarget'
-    Properties:
-      SubnetId: !Ref WSO2APIMPrivateSubnet2
-      FileSystemId: !Ref WSO2APIMEFSFileSystem
-      SecurityGroups:
-        - !Ref WSO2APIMEFSSecurityGroup
-  WSO2APIMAnalyticsEFSMountTarget:
-    Type: 'AWS::EFS::MountTarget'
-    Properties:
-      SubnetId: !Ref WSO2APIMPrivateSubnet1
-      FileSystemId: !Ref WSO2APIMAnalyticsEFSFileSystem
-      SecurityGroups:
-        - !Ref WSO2APIMEFSSecurityGroup
-  WSO2APIMAnalyticsAutoScalingGroup:
-    Type: 'AWS::AutoScaling::AutoScalingGroup'
-    Properties:
-      LaunchConfigurationName: !Ref WSO2APIMAnalyticsLaunchConfiguration
-      DesiredCapacity: 1
-      MinSize: 1
-      MaxSize: 1
-      LoadBalancerNames:
-        - !Ref WSO2APIMAnalyticsLoadBalancer
-      VPCZoneIdentifier:
-        - !Ref WSO2APIMPublicSubnet1
       Tags:
         - Key: Name
-          Value: WSO2APIM-Analytics-Instance
-          PropagateAtLaunch: 'true'
-        - Key: Product
-          Value: WSO2APIM-Analytics
-          PropagateAtLaunch: 'true'
-        - Key: Version
-          Value: 2.1.0
-          PropagateAtLaunch: 'true'
-        - Key: Cluster
-          Value: WSO2APIMAnalytics210SIMPLE
+          Value: WSO2APIMInstance2
           PropagateAtLaunch: 'true'
     DependsOn:
-      - WSO2APIMAnalyticsLaunchConfiguration
-      - WSO2APIMAnalyticsLoadBalancer
       - WSO2APIMLoadBalancer
-  WSO2APIMAnalyticsLaunchConfiguration:
+      - WSO2APIMNode2LaunchConfiguration
+  WSO2APIMAnalyticsSecurityGroup:
+    Type: 'AWS::EC2::SecurityGroup'
+    Properties:
+      VpcId: !Ref WSO2APIMVPC
+      GroupDescription: WSO2 API Manager Analytics Security Group
+      SecurityGroupIngress:
+        - IpProtocol: tcp
+          FromPort: '9763'
+          ToPort: '9763'
+          SourceSecurityGroupId: !Ref WSO2APIMAnalyticsLoadBalancerSecurityGroup
+        - IpProtocol: tcp
+          FromPort: '9443'
+          ToPort: '9443'
+          SourceSecurityGroupId: !Ref WSO2APIMAnalyticsLoadBalancerSecurityGroup
```

Comment:
Do we need to expose these as APIM doesn't have DAS based dashboards?

Reply:
It was required to expose the APIM Analytics Management Console if any user required to make any changes, hence port 9443 was exposed through the load balancer.

Reply:
Sounds good!